src/components/SearchAndFilter/FilterPanelSection/FilterPanelSection.js

```diff
@@ -23,50 +29,92 @@ const FilterPanelSection = ({ data, addToSelected, searchData }) => {
     setOverflowCounter(overflowCount);
   };
 
+  // Check if search term characters matches any characters in panel heading
+  const searchTermInHeading = hightlightSubString(heading, searchTerm).includes(
+    "<strong>"
```

Comment:
This tightly couples the expected result of `hightlightSubString`. I think a simple `string.contains` or common function used by both places would be better here as you might change `strong` to `<i>` in the future and this would result in unexpected failures here and other places that use it.